docs/LifecycleRulePost.md

```diff
@@ -0,0 +1,13 @@
+# LifecycleRulePost
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**bucket** | [**Reference**](Reference.md) | The bucket name for lifecycle rule creation. | [optional] 
+**rule_id** | **str** | Identifier for the rule that is unique to the bucket that it applies to. Can have a maximum length of 255 characters. If not specified, an id unique to the bucket will be generated in the format &#x60;fbRuleId&lt;number&gt;&#x60; where number increments, starting at 1. | [optional] 
+**keep_previous_version_for** | **int** | Time after which previous versions will be marked expired. Measured in milliseconds. Must be a multiple of 86400000 to represent a whole number of days. | [optional] 
```

Comment:
This field shouldn't be optional. Has this been generated from somewhere else?

Reply:
Generated from the staging branch for this version of the SDK. 
I think the issue here is that `optional` is the default for properties/parameters. Looks like we'll need to flag these as `required: true` and regenerate.